pkg/core/data/ipc/transactions/transaction.go

```diff
@@ -25,28 +24,21 @@ const (
 	Tx TxType = iota
 	// Distribute indicates the coinbase and reward distribution contract call.
 	Distribute
```

Comment:
In rusk Distribute=0 and Transfer=1. Just make sure to handle it properly

Reply:
Please report an issue with reference to document or rust code file where this can be confirmed.